packages/types/src/coin.ts

```diff
@@ -0,0 +1,6 @@
+import type { BNInput } from 'fuels';
+
+export type Coin = {
```

Comment:
Yes, in this case, we want a coin quantity-like type without all the other data in the AssetAmount.

Reply:
It was previously a type AssetWithDetails. I change it because the idea of the centralization is to externalize the types that the FuelWeb3 SDK will consume. The idea for change is because `getAssets`  will return an Asset list with all information, making easier for DApps to have assets with icons, names, and other things without needing an external source. And `getCoins` will return the coins the user has amount, asset_id, and probably UTXO.